man/org.freedesktop.import1.xml

```diff
@@ -0,0 +1,279 @@
+              in  b force,
+              out u transfer_id,
+              out o transfer_path);
+      ListTransfers(out a(usssdo) transfers);
+      CancelTransfer(in  u transfer_id);
+    signals:
+      TransferNew(u transfer_id,
+                  o transfer_path);
+      TransferRemoved(u transfer_id,
+                      o transfer_path,
+                      s result);
+  };
+  interface org.freedesktop.DBus.Peer { ... };
+  interface org.freedesktop.DBus.Introspectable { ... };
+  interface org.freedesktop.DBus.Properties { ... };
+};
+    </programlisting>
+
+    <!--method ImportFileSystem is not documented!-->
+
+    <refsect2>
+      <title>Methods</title>
+
+      <para><function>ImportTar()</function> and <function>ImportRaw()</function> import a system image and
+      place it into <filename>/var/lib/machines/</filename>. The first argument should be a file descriptor
+      (opened for reading) referring to the tar or raw file to import. It should reference either a file on
+      disk or a pipe or socket. When <function>ImportTar()</function> is used the file descriptor should
+      refer to a tar file, optionally compressed with
+      <citerefentry project="die-net"><refentrytitle>gzip</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+      <citerefentry project="die-net"><refentrytitle>bzip2</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+      or
+      <citerefentry project="die-net"><refentrytitle>xz</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
+      <command>systemd-importd</command> will detect the used compression scheme (if any) automatically. When
+      <function>ImportRaw()</function> is used the file descriptor should refer to a raw or qcow2 disk image
+      containing an MBR or GPT disk label, also optionally compressed with gzip, bzip2 or xz. In either case,
+      if the file is specified as file descriptor on disk, progress information is generated for the import
+      operation (since the size on disk is known then), if a socket or pipe is specified this is not
+      available. The file descriptor argument is followed by a local name for the image. This should be a
+      name suitable as a hostname and will be used to name the imported image below
+      <filename>/var/lib/machines</filename>. A tar import is placed as directory tree or
+      <citerefentry project="man-pages"><refentrytitle>btrfs</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      subvolume below <filename>/var/lib/machines/</filename>, under the name specified with no suffix
+      appended. A raw import is placed as file in <filename>/var/lib/machines/</filename> with the
+      <filename>.raw</filename> suffix appended. The <option>force</option> argument controls whether any
+      pre-existing image with the same name shall be removed for the operation. If true, it is removed, if
+      false the operation fails on a name conflict. Finally, the <option>read_only</option> argument controls
```

Comment:
```suggestion
      <filename>.raw</filename> suffix appended. If the <option>force</option> argument is true, any
      pre-existing image with the same name is removed before starting the operation. Otherwise, the 
      operation fails if an image with the same name already exists. Finally, the
      <option>read_only</option> argument controls
```